.github/workflows/interface-unit-tests.yml

```diff
@@ -102,6 +123,18 @@ jobs:
             "device-tests": 2
           }
           EOF
+          elif [ "${{ inputs.python_warning_level }}" == "error" ];
+          then
+            cat >matrix_max_parallel.json <<-EOF
+          {
+            "default": 1,
+            "core-tests": 1,
+            "gradients-tests": 1,
+            "jax-tests": 1,
+            "tf-tests": 1,
+            "device-tests": 1
+          }
+          EOF
```

Comment:
```suggestion
          elif [ "${{ inputs.python_warning_level }}" == "error" ];
          then
            cat >matrix_max_parallel.json <<-EOF
          {
            "default": 1
          }
          EOF
```

Reply:
Are these all 1 by default then if not specified?